components/worktemplates/components/modal_body.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I think this file is not needed at all. It's only used for this page preview page so we might as well put everything in the page.